plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/gettingstarted/editor/GettingStartedTelemetryUtils.kt

```diff
@@ -70,6 +70,37 @@ fun getEnabledConnectionsForTelemetry(project: Project?): Set<AuthFormId> {
 fun getEnabledConnections(project: Project?): String =
     getEnabledConnectionsForTelemetry(project).joinToString(",")
 
+fun getAuthScopesForTelemetry(project: Project?): Set<String> {
+    project ?: return emptySet()
+    val scopes = mutableSetOf<String>()
+
+    fun addScopes(connection: ActiveConnection) {
+        if (connection !is ActiveConnection.NotConnected) {
+            val connectionScopes = connection.activeConnectionBearer?.scopes
+            if (connectionScopes != null) {
+                scopes.addAll(connectionScopes)
+            }
+        }
+    }
+
+    val explorerConnection = checkIamConnectionValidity(project)
```

Comment:
We could also probably check the connection type and use the addScopes function only if the connection type is IDC. We can probably just hardcode it anyway

Reply:
Check the connection type of the explorer only? Would that not ignore the other connections/scopes available if those other features are using the IdC connection type while the explorer isnt?